### ISSUE TITLE
pallete prop is misspelled

### ISSUE THREAD

**Issue by tuantle** [capitalone/react-native-pathjs-charts#175]
Please rename to palette.

**Comment by marzolfb** [comment]
Agreed this is a bug. Just for historical context, this library is based off of https://github.com/rsamec/react-pathjs-chart and that seems to be where the misspelling originated.

While introducing a new property with the correct spelling is straightforward and non-controversial,  removing the old misspelled version of the property is arguable as doing so would be introducing a breaking change. It would be interesting to hear from others on whether a breaking change like that would be appropriate.
